packages/did-comm/src/__tests__/coordinate-mediation-message-handler.test.ts

```diff
@@ -83,11 +90,9 @@ describe('coordinate-mediation-message-handler', () => {
             ...new FakeDidResolver(() => agent).getDidFakeResolver(),
           }),
         }),
-        // @ts-ignore
         new DIDComm([new DIDCommHttpTransport()]),
```

Comment:
As consequence to changing parameter of DIDComm to an object.
```suggestion
        new DIDComm({ transports: [new DIDCommHttpTransport()]}),
```

Reply:
Cheers, updated (using your patch)

Reply:
#resolved